core/vm/evm.go

```diff
@@ -266,6 +317,19 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 			contract.UseGas(contract.Gas)
 		}
 	}
+
+	if UsingOVM {
+		// OVM_ENABLED
+		if isTarget {
+			evm.OriginalTargetResult = ret[96:]
```

Comment:
Ignore the first 96 bytes (the return status and ABI encoding parameters).

Reply:
Is there any chance `len(ret) < 96`?

Reply:
Yes -- fixed to address this.